docker/Dockerfile

```diff
@@ -316,8 +316,14 @@ RUN  \
         curl -sLO https://ffmpeg.org/releases/ffmpeg-${FFMPEG_VERSION}.tar.bz2 && \
         tar -jx --strip-components=1 -f ffmpeg-${FFMPEG_VERSION}.tar.bz2
 
+COPY debian/patches/*.patch /tmp/ffmpeg/
+
 RUN \
         DIR=/tmp/ffmpeg && mkdir -p ${DIR} && cd ${DIR} && \
+        for patch in *.patch; do \
+            filename="$( grep '+++' ${patch} | awk '{ print $2 }' | sed 's|jellyfin-ffmpeg/||' )"; \
+            patch --verbose ${filename} ${patch}; \
```

Comment:
Heh, why not just `patch -p1 < "$patch"`?

Reply:
For whatever reason, the patch command wasn't liking the Debian patch format, mostly because the actual path to the file was different (we're one level lower at this stage). This makes it super explicit.

Reply:
But that is what `-pN` does - it drops first `N` dirs in a path before applying the patch...

Reply:
Also your method would fail for multi-file patches...

Reply:
Oh really? OK, yea, that's nicer. I went through the man page quickly and didn't see the obvious solution. I guess that's an improvement to be made if we implement any more patches ;-) Hopefully we don't and this is redundant before then anyways.